data/reusables/files/delete_branch.md

```diff
@@ -0,0 +1 @@
+1. Optionally, [delete the branch](/articles/deleting-unused-branches). This keeps the list of branches in your repository tidy.
```

Comment:
```suggestion
1. Optionally, delete the branch. For more information, see "(/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository)."
```

This is a small edit to remove the inline link. Some of our older documentation uses inline links, but our current style guide requires links to follow the format of `For more information, see "Article."`